downstream/modules/platform/proc-deploy-eda-controller-with-aap-operator-ocp.adoc

```diff
@@ -35,7 +35,7 @@ extra_settings:
     value: '12'
 ----
 +
-. Click btn[Reload] and btn[Save]. Return to the Form view.
+. Click btn:[Reload] and btn:[Save]. Return to the Form view.
```

Comment:
Are we supposed to use *Form* here? I'm not sure.

Reply:
"Form view" is the name of the radio button you can select, the other view they are currently in is "YAML view"

Reply:
I  phrased that wrong: what I meant to ask was whether **Form** was supposed to be bold for a UI element - I wasn't questioning the word "form"